src/patterns/gen-ai/aws-model-deployment-sagemaker/README_hugging_face.md

```diff
@@ -85,30 +77,8 @@ Parameters
 - id string
 - props IHuggingFaceSageMakerEndpointProps
 
```

Comment:
HuggingFaceSageMakerEndpoint initializer is missing.

Reply:
done in [e8ebbf5](https://github.com/awslabs/generative-ai-cdk-constructs/pull/180/commits/e8ebbf58d89f9699465624722f45c3eb9401228f)